### PR TITLE
Convert to monorepo structure

### PR DESCRIPTION
**Pull request by njfamirm**
## Description

Convert to monorepo structure

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: <https://github.com/shaf-i/alshaf-i-pwa/blob/main/CONTRIBUTING.md#commit-message-format>
- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Comment by njfamirm** [comment]
ببین چطوره
وقت انتقامه 🤣

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -24,4 +30,4 @@ jobs:
           cache: 'yarn'
 
       - run: yarn install --frozen-lockfile
-      - run: yarn lint
+      - run: yarn format
```
Comment (MM25Zamanian) [inline .github/workflows/lint.yml:33]: ???

File: package.json [hunk 2]
```diff
@@ -3,62 +3,36 @@
   "version": "0.0.0",
   "description": "Alshaf-i PWA",
   "author": "MohammadMahdi Zamanian <mm25zamanian@gmail.com> (https://mm25zamanian.ir)",
+  "contributes": [
+    "MohammadMahdi Zamanian <mm25zamanian@gmail.com> (https://mm25zamanian.ir)",
```
Comment (MM25Zamanian) [inline package.json:7]: Please Remove This Line

File: packages/alshaf-i-content/index.html [hunk 3]
```diff
@@ -57,7 +57,7 @@
   <body>
     <app-index></app-index>
 
-    <script type="module" src="src/app-index.js"></script>
+    <script type="module" src="dist/app-index.js"></script>
```
Comment (MM25Zamanian) [inline packages/alshaf-i-content/index.html:60]: ???

File: packages/alshaf-i-content/package.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "alshaf-i-pwa-content",
+  "version": "0.0.0",
+  "description": "Alshaf-i PWA content",
+  "author": "MohammadMahdi Zamanian <mm25zamanian@gmail.com> (https://mm25zamanian.ir)",
+  "contributes": [
+    "MohammadMahdi Zamanian <mm25zamanian@gmail.com> (https://mm25zamanian.ir)",
```
Comment (MM25Zamanian) [inline packages/alshaf-i-content/package.json:7]: Remove This Line ?

**Review by MM25Zamanian — APPROVED** [review]

**Comment by MM25Zamanian** [comment]
چرا `alshaf-i-content` ???

**Comment by njfamirm** [comment]
> چرا `alshaf-i-content` ???

چون lit-dev-content

**Comment by njfamirm** [comment]
بیشتر از دو ساعت نمیتونم صبر کنم برای پاسخ دادن به وویس

**Pull request merged**